src/Symfony/Component/Notifier/Bridge/Sinch/CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 CHANGELOG
 =========
 
-5.1
+5.1.0
```

Comment:
We don't add new features in patch version, so 5.1 is correct (and we should probably NOT fix the existing changelogs :)).

Reply:
OK all others are using `5.1.0`, thats why I changed it